Sources/StreamChatUI/CommonViews/PlayerView.swift

```diff
@@ -7,6 +7,7 @@ import StreamChat
 import UIKit
 
 /// A view that shows a playing video content.
+@objc(StreamPlayerView)
```

Comment:
@martinmitrevski We forgot the changelog

Reply:
do we need it here? I can raise another PR if you think so.

Reply:
I think it makes sense, it fixed a problem for a customer, so more customers can have the same issue